packages/react-button/src/components/SplitButton/SplitButton.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { SplitButton } from './SplitButton';
+import { isConformant } from '../../common/isConformant';
+
+describe('SplitButton', () => {
+  isConformant({
+    Component: SplitButton,
+    displayName: 'SplitButton',
+    disabledTests: ['has-docblock', 'as-renders-html', 'as-passes-as-value', 'as-renders-react-class', 'as-renders-fc'],
```

Comment:
Remove these?

Reply:
Unless I'm misunderstanding and the as tests work for the other buttons but not this one